virttest/env_process.py

```diff
@@ -314,6 +314,15 @@ def check_image(test, params, image_name, vm_process_status=None):
                               " Skip the image check.")
                 check_image_flag = False
 
+    # Save the potential bad image when the test is not passed.
+    # It should before image check.
+    if params.get("save_image", "no") == "yes":
```

Comment:
Would you please also add log in case `vm_process_status != "dead"` saying something like "Not saving images, VM is still running."?

Reply:
updated, thanks.